src/components/Navbar/Auth/PasswordRecoveryFinishForm.spec.js

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+
+import { submitFinishPasswordRecover, verifyPasswordRecoveryToken } from "../../../services/auth";
+
+import { fireEvent, act, renderWithStoreAndTheme, waitFor } from "../../../test-utils";
+import Constants from "../../../utils/Constants";
+import { SnackbarProvider } from "notistack";
+import { BrowserRouter } from "react-router-dom";
+import HomePage from "../../../pages/HomePage";
+import Notifier from "../../Notifications/Notifier";
+import AppTheme from "../../../AppTheme";
+import Navbar from "..";
+
+jest.mock("../../../services/auth");
+jest.mock("react-router-dom", () => {
+    const original = jest.requireActual("react-router-dom");
+    return {
+        ...original,
+        useParams: jest.fn().mockReturnValue({ token: "test123" }),
+    };
+});
+
```

Comment:
In this function why did you mock `useParams` ? I understand that this and other `react-router` functions are not defined in JSDom but if you are using `BrowserRouter`s to wrap your test components the mock should not be necessary, unless I'm overlooking something important.